packages/mobile/src/backup/DelayButton.tsx

```diff
@@ -0,0 +1,54 @@
+import colors from '@celo/react-components/styles/colors.v2'
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, View } from 'react-native'
+import { useDispatch, useSelector } from 'react-redux'
+import { setBackupDelayed } from 'src/account/actions'
+import CeloAnalytics from 'src/analytics/CeloAnalytics'
+import { CustomEventNames } from 'src/analytics/constants'
+import { Namespaces } from 'src/i18n'
+import { navigateBack } from 'src/navigator/NavigationService'
+import TopBarButton from 'src/navigator/TopBarButton.v2'
+import { RootState } from 'src/redux/reducers'
+import { isBackupTooLate } from 'src/redux/selectors'
+
+interface DelayStateProps {
+  backupTooLate: boolean
+  backupDelayedTime: number
+}
+
+const mapStateToHeaderProps = (state: RootState): DelayStateProps => {
+  return {
+    backupTooLate: isBackupTooLate(state),
+    backupDelayedTime: state.account.backupDelayedTime,
+  }
+}
+
+export default function DelayButton() {
+  const { backupTooLate, backupDelayedTime } = useSelector(mapStateToHeaderProps)
```

Comment:
nit: `useSelector` has a subtle difference compared to `connect`, it uses `===` comparison to decide if it needs re-rendering. Here since the `mapStateToProps` is returning a new object it will always re-render whenever the redux state changes. See https://react-redux.js.org/api/hooks#equality-comparisons-and-updates

```suggestion
  const { backupTooLate, backupDelayedTime } = useSelector(mapStateToHeaderProps, shallowEqual)
```